pkg/agent/openflow/network_policy.go

```diff
@@ -1440,12 +1439,13 @@ func parseMetricFlow(flow string) (uint32, types.RuleMetric) {
 
 func (c *client) NetworkPolicyMetrics() map[uint32]*types.RuleMetric {
 	result := map[uint32]*types.RuleMetric{}
-	ovsctlClient := ovsctl.NewClient(c.nodeConfig.OVSBridge)
-	egressFlows, _ := ovsctlClient.DumpTableFlows(uint8(EgressMetricTable))
-	ingressFlows, _ := ovsctlClient.DumpTableFlows(uint8(IngressMetricTable))
+	egressFlows, _ := c.ovsctlClient.DumpTableFlows(uint8(EgressMetricTable))
+	ingressFlows, _ := c.ovsctlClient.DumpTableFlows(uint8(IngressMetricTable))
 	collectMetricsFromFlows := func(flows []string) {
 		for _, flow := range flows {
-			if strings.Contains(flow, "priority=0") {
+			// There could be other flows like default flow and traceflow flows in the table.
+			// Only metrics flows are supposed to have priority 200.
+			if !strings.Contains(flow, "priority=200,") {
```

Comment:
Probably better to use "priorityNormal".

Reply:
Done, thanks.